cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -133,11 +133,13 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
   @Test def list_indexes() {
     prepareAndTestQuery(
       title = "List indexes",
-      text = "Calling the built-in procedure `db.indexes` will list all indexes, including their names.",
```

Comment:
When discussing if the procedures should be rewritten to use the new commands we found that it lead to errors on `AUTH_DISABLED` and abandoned it. It was also discussed that suddenly needing privileges you didn't before might be a breaking change for the procedure (since it would not be granted by default to `PUBLIC`, like the `EXECUTE`). So we decided to keep the old implementation and not apply the new privileges to the procedure at all.

Reply:
We should however add a comment about it, either in the listing indexes/constraints section or if there is some other place where the procedures are documented.